### PR TITLE
queryset.go: added getDBObject method

### PR DESCRIPTION
**Pull request by rajatgpt1521**
- This method returns DB object w.r.t queryset used to call it.
- The returned object can further be used to call gorm related
property for many to many relation.

### CODE REVIEW

File: examples/comparison/gorm4/autogenerated_gorm4.go [hunk 1]
```diff
@@ -153,25 +159,25 @@ func (qs UserQuerySet) GetUpdater() UserUpdater {
 
 // IDEq is an autogenerated method
 // nolint: dupl
-func (qs UserQuerySet) IDEq(ID uint) UserQuerySet {
+func (qs UserQuerySet) IDEq(ID uint64) UserQuerySet {
```
Comment (jirfag) [inline examples/comparison/gorm4/autogenerated_gorm4.go]: why did you change `uint` to `uint64` everywhere? it's the incompatible change

Comment (rajatgpt1521) [reply examples/comparison/gorm4/autogenerated_gorm4.go]: I didn't did anything for above mentioned change. I am not able to figure out the root cause of it. Can we start the discussion in order to resolve this issue?

**Comment by jirfag** [comment]
I've supported `GetDB` method in the master branch.
